src/main.java

```diff
@@ -0,0 +1,22 @@
+package src;
+public class main {
+    public static void  main(String[] args){
+
+        FirePokemon charmender = new FirePokemon("Charmender", 10,200,50,"none");
```

Comment:
"none" is niet zo'n goede optie voor je second type. Je kunt misschien `null` invullen, maar beter nog maak je een tweede constructor zonder deze waarde.